Data/datasetloader.py

```diff
@@ -1 +1,58 @@
-# work on pytroch dataset loader -- Natalie
\ No newline at end of file
+# work on pytroch dataset loader -- Natalie
+#inspired by Kaggle: https://www.kaggle.com/code/mdteach/torch-data-loader-flicker-8k/notebook
+
+
+import torch
+from torch.utils.data import Dataset, DataLoader
+import pandas as pd
+import torchvision.transform as T
+from .vocab import Vocabulary
+import os
+from PIL import Image
+
+
+class Flickr8Dataset(Dataset):
+    '''
+    root_dir: flickr dataset folder/Images
+    captions_file: flickr dataset location/captions.txt
+    '''
+    def __init__(self, root_dir, captions_file,
+                 transform=T.Compose([T.Resize((224,224)),T.ToTensor()]),
```

Comment:
can we use the transform in the imageEncoder? or do the images go through 2 transforms.

Reply:
I think we can! I'll update my code to reflect that.